.claude/CLAUDE.md

```
@@ -0,0 +1,59 @@
+
+## 빌드 & 실행
+
+```bash
+./gradlew build      # 빌드 (flyway validate 포함)
+./gradlew test       # 테스트
+./gradlew bootRun    # 로컬 실행 (Docker Compose로 PostgreSQL, Redis 필요)
+```
+
+## 프로젝트 구조
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)